otelcollector/scripts/main.sh

```diff
@@ -286,27 +294,26 @@ echo_var "OTELCOLLECTOR_VERSION" "$OTELCOLLECTOR_VERSION"
 PROMETHEUS_VERSION=`cat /opt/microsoft/otelcollector/PROMETHEUS_VERSION`
 echo_var "PROMETHEUS_VERSION" "$PROMETHEUS_VERSION"
 
-#get ruby version
-RUBY_VERSION=`ruby --version`
-echo_var "RUBY_VERSION" "$RUBY_VERSION"
-
-echo "Starting telegraf"
+echo "starting telegraf"
 if [ "$TELEMETRY_DISABLED" != "true" ]; then
-  /opt/telegraf/telegraf --config /opt/telegraf/telegraf-prometheus-collector.conf &
-  TELEGRAF_VERSION=`/opt/telegraf/telegraf --version`
+  telegraf --config /opt/telegraf/telegraf-prometheus-collector.conf &
```

Comment:
doesn't mdsd drop telegraf as well? Isn't better to point to the right telegraf's path when starting it ?

Reply:
Yes right now we are removing it though in setup.sh. I will add the full path just in case though, good point